app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -288,6 +290,18 @@ def create_vm(args = {})
       vm
     end
 
+    def id_name_substitution(argument_list, argument)
```

Comment:
This does not substitute the name and id, it gets the id or finds the id from the name.

Reply:
True, maybe I can call it get_id_from_name. Do you think its a more suitable name?

Reply:
I just realized that because of the next comment I don't need a new name, this line will be in the get_ovirt_id function and not in a different method.